### ISSUE TITLE
Source file copying fails since upgrade to 1.1.2

### ISSUE THREAD

**Issue by plnordquist** [jenkinsci/code-coverage-api-plugin#140]
# Environment
Jenkins 2.204.1 LTS
Code Coverage API Plugin 1.1.2
Maven 3.6.3
jacoco-maven-plugin 0.8.4
Spring Boot 2.1.8

# Issue

I'm having issues with this plugin and source file copying since I upgraded to 1.1.2.  The exception I'm seeing is below, it's trying to grab files from after a maven build using jacoco-maven-plugin while building a Spring Boot project.

```
Starting copy source file <file>.java. 
java.io.IOException: Unable to find source file <file>.java in workspace /var/lib/jenkins/jenkins-home/workspace/<multibranch-workspace>
	at io.jenkins.plugins.coverage.source.DefaultSourceFileResolver$SourceFilePainter.invoke(DefaultSourceFileResolver.java:147)
	at io.jenkins.plugins.coverage.source.DefaultSourceFileResolver$SourceFilePainter.invoke(DefaultSourceFileResolver.java:127)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3069)
	at hudson.remoting.UserRequest.perform(UserRequest.java:211)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
	Suppressed: hudson.remoting.Channel$CallSiteStackTrace: Remote call to <node-name>
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1741)
		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)
		at hudson.remoting.Channel.call(Channel.java:955)
		at hudson.FilePath.act(FilePath.java:1069)
		at hudson.FilePath.act(FilePath.java:1058)
		at io.jenkins.plugins.coverage.source.DefaultSourceFileResolver.lambda$resolveSourceFiles$0(DefaultSourceFileResolver.java:92)
		at java.util.HashMap.forEach(HashMap.java:1289)
		at io.jenkins.plugins.coverage.source.DefaultSourceFileResolver.resolveSourceFiles(DefaultSourceFileResolver.java:81)
		at io.jenkins.plugins.coverage.CoverageProcessor.performCoverageReport(CoverageProcessor.java:117)
		at io.jenkins.plugins.coverage.CoveragePublisher.perform(CoveragePublisher.java:96)
		at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:80)
		at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:67)
		at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
		at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
		... 4 more
```

**Comment by smithhannahm** [comment]
It looks like PR #136 addresses this

**Comment by alexeyulasevich** [comment]
Does somebody know any workaround?

**Comment by bgreen1005** [comment]
I've created a PR to fix this issue :) https://github.com/jenkinsci/code-coverage-api-plugin/pull/142

**Comment by bgreen1005** [comment]
I've created a PR to fix this issue :) https://github.com/jenkinsci/code-coverage-api-plugin/pull/142

**Comment by cizezsy** [comment]
This should be fixed in version 1.14. Thanks very much for @bgreen1005 's help!
